fearless-utils/src/test/java/jp/co/soramitsu/fearless_utils/encrypt/seed/ethereum/EthereumSeedFactoryTest.kt

```diff
@@ -0,0 +1,43 @@
+package jp.co.soramitsu.fearless_utils.encrypt.seed.ethereum
+
+import com.google.gson.Gson
+import jp.co.soramitsu.fearless_utils.encrypt.keypair.ethereum.EthereumKeypairFactory
+import jp.co.soramitsu.fearless_utils.extensions.toHexString
+import jp.co.soramitsu.fearless_utils.getResourceReader
+import jp.co.soramitsu.fearless_utils.encrypt.junction.BIP32JunctionDecoder
+import jp.co.soramitsu.fearless_utils.encrypt.mnemonic.MnemonicTestCase
+import org.junit.Assert
+import org.junit.Test
+
+class EthereumSeedFactoryTest {
```

Comment:
Looks like it is not exactly seed factory test — it is keypair generator test. Maybe, it's better give it more concise name

Reply:
Done